src/app/(dashboard)/overview/dashboardInsights.tsx

```diff
@@ -0,0 +1,64 @@
+import { Icons } from "@/components/icons";
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { OverviewDto } from "@/app/api/teams/types/overview.dto";
+
+interface DashboardInsightsProps {
+  overview?: OverviewDto;
+}
+
+export function DashboardInsights({ overview }: DashboardInsightsProps) {
+  return (
+    <div className="grid gap-4 md:grid-cols-2 lg:grid-cols-3">
+      <Card>
+        <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
+          <CardTitle className="text-sm font-medium">
+            Total CRED earned
+          </CardTitle>
+          <Icons.pizza></Icons.pizza>
+        </CardHeader>
+        <CardContent>
+          <div className="text-2xl font-bold">
+            {overview?.totalScore.toFixed(2)}
+          </div>
+          <p className="pt-1 text-xs text-muted-foreground">
+            +{overview?.weekGrowth.toFixed(2)}% CRED earned
+          </p>
+        </CardContent>
+      </Card>
+
+      <Card>
+        <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
+          <CardTitle className="text-sm font-medium">
+            CRED earned this week
+          </CardTitle>
+          <Icons.gitBranch></Icons.gitBranch>
+        </CardHeader>
+        <CardContent>
+          <div className="text-2xl font-bold">
+            {overview?.weekScore.toFixed(2)}
+          </div>
+          <p className="pt-1 text-xs text-muted-foreground">
+            +{overview?.wowScore.toFixed(2)}% over last week
+          </p>
+        </CardContent>
+      </Card>
+
+      <Card>
+        <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
+          <CardTitle className="text-sm font-medium">
+            Total contributors
+          </CardTitle>
+          <Icons.users></Icons.users>
+        </CardHeader>
+        <CardContent>
+          <div className="text-2xl font-bold">
+            {overview?.totalContributors}
+          </div>
+          <p className="pt-1 text-xs text-muted-foreground">
+            Contributors on team
+          </p>
+        </CardContent>
+      </Card>
+    </div>
+  );
+}
```

Comment:
looks good but this component should be on /src/components/something instead of here